src/es/components/molecules/teaser/Teaser.js

```diff
@@ -88,7 +88,7 @@ export default class Teaser extends Intersection() {
     if (this.getAttribute('namespace') === 'teaser-overlay-' && this.aArrow) this.aArrow.setAttribute('hover-set-by-outside', '')
     this.css = /* css */`
       :host {
-        max-width: 100%;
+        max-width: 100%;        
```

Comment:
essential spaces :-)
maybe you run npm run fix for the linter, that will adjust those...

Reply:
i will in the future, sorry for that!